website/docs/getting-started/Installation.md

```diff
@@ -62,8 +62,8 @@ author_url: https://rohit.page
              // Change with your site colors
              primaryColor: '#1890ff',
            },
-         },
-       ] satisfies Redocusaurus.PresetEntry,
```

Comment:
this is correct, it's used in the repo also - https://github.com/rohit-gohri/redocusaurus/blob/42fc3cb7f95a928b6b0c4f175a9e953868ec6ee8/website/docusaurus.config.ts#L65

 could you share your repo?